src/ssl.c

```diff
@@ -17702,6 +17702,106 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
     }
     #endif /* !defined(HAVE_FIPS) || (defined(HAVE_FIPS_VERSION) && \
               (HAVE_FIPS_VERSION > 2)) */
+
+#if !defined(HAVE_FIPS) && !defined(HAVE_SELFTEST)
+#if !defined(WOLFSSL_NOSHA512_224)
```

Comment:
Is `WOLFSSL_NOSHA512_224` a new define? I'm not seeing it in the master branch of the wolfSSL repo, nor am I seeing it defined in this PR.

Reply:
Yes, WOLFSSL_NOSHA512_224 and WOLFSSL_NOSHA512_256 macros are newly introduced.